homeassistant/components/weatherflow_cloud/strings.json

```diff
@@ -56,6 +56,34 @@
       "lightning_strike_last_epoch": {
         "name": "Lightning last strike"
       },
+      "precip_accum_last_1hr": {
+        "name": "Rain last hour"
+      },
+
+      "precip_accum_local_day": {
+        "name": "Precipitation today"
+      },
+      "precip_accum_local_day_final": {
+        "name": "Nearcast precipitation today"
```

Comment:
what happens with this sensor if the user disables nearcast data?

Reply:
Apparently nothing. 

The toggle says 
<img width="315" height="60" alt="image" src="https://github.com/user-attachments/assets/94187e1b-f7ea-4d79-9a50-0878bea908d1" />

And when its off I still get the data over REST.

Reply:
Thats what I gather... :)